text/0000-contingency-plan.md

```diff
@@ -0,0 +1,425 @@
+- Feature Name: N/A
+- Start Date: 2015-05-07
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+This RFC has two main goals:
+
+- define what precisely constitutes a breaking change for the Rust language itself;
+- define a language versioning mechanism that extends the sorts of
+  changes we can make without causing compilation failures (for
+  example, adding new keywords).
+  
+# Motivation
+
+With the release of 1.0, we need to establish clear policy on what
+precisely constitutes a "minor" vs "major" change to the Rust language
+itself (as opposed to libraries, which are covered by [RFC 1105]).
+**This RFC proposes limiting breaking changes to changes with
+soundness implications**: this includes both bug fixes in the compiler
```

Comment:
@nrc Right, I mean that until 2.0, we limit breaking changes to bugs/soundness issues. I can clarify the language here.

@ardentsonata I think this same policy would apply to version 2.0 and beyond, though of course we could change it. That is, we will continue to fix bugs and soundness issues. I am basically trying to define more precisely what can go into a "minor change" in Rust (whether it be from 1.1 to 1.2, or 2.1 to 2.2).